pkg/sql/colexec/auto_incr.go

```diff
@@ -27,6 +26,7 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/sql/errors"
 	"github.com/matrixorigin/matrixone/pkg/vm/engine"
 	"github.com/matrixorigin/matrixone/pkg/vm/process"
+	"golang.org/x/exp/constraints"
 )
 
 var AUTO_INCR_TABLE = "%!%mo_increment_columns"
```

Comment:
What is this?  AUTO_INCR_TABLE ...

Reply:
this is the hide table which store the auto_incr col value